plugins/listblock/plugin.js

```diff
@@ -83,8 +84,11 @@ CKEDITOR.plugins.add( 'listblock', {
 			},
 
 			proto: {
-				add: function( value, html, title ) {
-					var id = CKEDITOR.tools.getNextId();
+				add: function( value, html, title, language, editor ) {
```

Comment:
Five parameters are probably too many.

I'm wondering if we could replace the added ones with the `options` object pattern:

```suggestion
				add: function( value, html, title, options ) {
				
					options = CKEDITOR.object.merge( someDefaultOptions, options );
```

The second idea I had: maybe we could pass the editor to the list block constructor and save it as `this._.editor`? However, I'm not sure if a list block instance can't be reused in another editor :thinking:

I see even two more solutions, probably even easier ones:

* pass only possible language fallbacks (so basically `editor.langCode`) instead of the whole editor,
* add the `[lang]` attribute only if it's known; it's absence should make the screen reader use the nearest available value – so the editor's container `[lang]` attribute.

Reply:
Thanks for the ideas 👍 
Let's go with applying the `lang` attribute when it's defined.